Tools/Cli-LoRa-Device-Provisioning/Keygen.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Cli_LoRa_Device_Provisioning
+{
+    using System;
+
+    public static class Keygen
+    {
+        public static string Generate(int keyLength)
+        {
+            var randomKey = string.Empty;
+            var rnd = new Random();
```

Comment:
you could keep the Random as a static on the Keygen class and reuse it.

Reply:
Great catch. Done.